src/main.rs

```diff
@@ -249,6 +249,39 @@ pub(crate) async fn run_encrypted_payload(
         }
     }
 
+    // Set execution permission for listed revocation actions
```

Comment:
The Python agent just warns when the name doesn't start with `local_action_` (and TBH I don't know what is the benefit of enforcing the scripts naming). About setting the permission, we could make it an error. The python agent don't have this problem (yet) because only python actions loaded as modules are supported.

Reply:
I also don't see any benefit to the `local_action_` prefix since it adds no information, I think this is another one of those things that the Python agent does for historical reasons and should be changed eventually. But I am fine with keeping it for now just to get the same behavior from the agents and just keeping that as a warning. 

If you are okay with it, throwing an error when the perm can't be set seems appropriate to me (maybe eventually it could be configurable by the user whether to have a warning or error, but that seems out of scope here).

Reply:
I'll change it to throw an error when the permissions can't be set.

Reply:
Modified to throw an error in case setting the permissions fail.